.github/workflows/package_pypi.yml

```diff
@@ -3,15 +3,31 @@ on:
     types: [published]
   schedule:
     - cron: '0 1 * * *'
+  workflow_dispatch:
```

Comment:
should we remove the manual trigger again?

Reply:
I think we should keep it so that we can trigger a quick test without having to wait for the next scheduled time.